aquamarine-vm/src/aquamarine_stepper_vm.rs

```diff
@@ -103,6 +135,49 @@ impl AquamarineVM {
 
         Ok(outcome)
     }
+
+    fn update_current_particle(&self, particle_id: String, init_user_id: String) {
+        let mut params = self.current_particle.lock();
+        params.particle_id = particle_id;
+        params.init_user_id = init_user_id;
+    }
+}
+
+fn prepare_args(
```

Comment:
Could you please also replace the same code from the 264th line with this function.